cabal-install/Distribution/Client/Get.hs

```diff
@@ -162,12 +163,16 @@ unpackPackage :: Verbosity -> FilePath -> PackageId
               -> PackageDescriptionOverride
               -> FilePath  -> IO ()
 unpackPackage verbosity prefix pkgid descOverride pkgPath = do
-    let pkgdirname = display pkgid
-        pkgdir     = prefix </> pkgdirname
-        pkgdir'    = addTrailingPathSeparator pkgdir
+    let pkgdirname               = display pkgid
+        pkgdir                   = prefix </> pkgdirname
+        pkgdir'                  = addTrailingPathSeparator pkgdir
+        emptyDirectory directory = null <$> listDirectory directory
```

Comment:
`listDirectory` is since `1.2.5.0`, tet we have `listDirectory` in `Distribution.Compat.Directory`.

Reply:
Hmm. this is `cabal-install` which is GHC-7.10+; yet `directory >=1.2.2.0` only.

I think we need to bump lower-bounds of boot libraries. @23Skidoo opinions?

Reply:
Ok, so, I will use `listDirectory` from there.

Reply:
For now, switch it to use compat module.

Reply:
I changed the import to use `listDirectory` from the `Distribution.Compat.Directory` module.

Reply:
Ddddddsssddddddddssddddddddddddrsdraaddasdasdaddddddsdsrssddsdrsd